ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/CancunTargetingGasLimitCalculator.java

```diff
@@ -85,4 +92,16 @@ public long getBlobGasPerBlob() {
   public long getTargetBlobGasPerBlock() {
     return targetBlobGasPerBlock;
   }
+
+  public long getMaxBlobGasPerBlock() {
```

Comment:
You can move these to Osaka.

Reply:
does it make sense to make these fields protected instead of using getters as for feeMarket?

Reply:
yes it does